lib/auth/vault.js

```diff
@@ -128,7 +144,7 @@ function authenticateV4Request(params, requestContexts, callback) {
             requestContext: serializedRCs,
         },
         (err, userInfo) => vaultSignatureCb(err, userInfo,
-                                            params.log, callback)
+            params.log, callback, streamingV4Params)
```

Comment:
This should be aligned with the first argument from the previous line and `streamingV4Params` should go to the next line.